packages/native-federation-tests/typedoc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": ["../../typedoc.base.json"],
+    "entryPoints": ["src/index.ts"],
```

Comment:
I think that [these entrypoints](https://github.com/module-federation/universe/blob/main/packages/native-federation-tests/tsup.config.ts#L9) are missing